### PR TITLE
Backporting height fixes in preparation for 2.8.6

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e6c71ca7ffaed1cfdbb53f0743c899c35031137e: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4521-bb9963190af7da9e1f64ff89404f375b0866bbfc-x86-android.apk)
</details>

File: src/qml/QFieldSettings.qml [hunk 1]
```diff
@@ -1117,7 +1117,7 @@ Page {
                   }
 
                   Label {
-                      text: qsTr( "Z values which are recorded from the positioning device will be corrected by this value. If a value of 1.6 is entered, this will result in a correction of -1.6 for each recorded value." )
+                      text: qsTr( "This value will correct the Z values recorded from the positioning device. If a value of 1.6 is entered, QField will automatically subtract 1.6 from each recorded value. Make sure to insert the effective antenna height, i.e. pole length + antenna phase centre offset." )
```
Comment (m-kuhn) [inline src/qml/QFieldSettings.qml:1120]: I would defer this change to the next release, while it improves English, it breaks all translations

**Comment by mbernasocchi** [comment]
@domi4484 is this all needed?

File: src/qml/QFieldSettings.qml (hunk 1 above)
Comment (nirvn) [reply src/qml/QFieldSettings.qml]: @m-kuhn , removed.

**Review by m-kuhn — APPROVED** [review]

**Comment by mbernasocchi** [comment]
Screenshot_20230821_081021
Screenshot_20230821_080951
Screenshot_20230821_080946_swisstopo

**Comment by mbernasocchi** [comment]
beta and dev using geoid correction :)
will test with imu later

**Comment by nirvn** [comment]
@mbernasocchi , sounds like we're ready for 2.8.6?

**Pull request merged**